.github/workflows/ci-actions.yml

```diff
@@ -69,19 +69,18 @@ jobs:
         uses: joschi/setup-jdk@68381f2c0646f942f70b69f8e81fe10e1ed5d293
         with:
           java-version: 11
-      - name: Compute cache restore key
-        # Always recompute on a push so that the maven repo doesnt grow indefinitely with old versions
+      - name: Get Date
+        id: get-date
         run: |
-          if ${{ github.event_name == 'pull_request' }}; then echo "::set-env name=COMPUTED_RESTORE_KEY::q2maven-"; fi
+          echo "::set-output name=date::$(/bin/date -u "+%Y-%m")"
+        shell: bash
       - name: Cache Maven Repository
         id: cache-maven
         uses: n1hility/cache@v2
```

Comment:
I think we can move back to `actions/cache@v2` since we're not using the `restore-keys` and `restore-only` keys anymore

Reply:
Done.